lib/main.js

```diff
@@ -8,6 +8,8 @@ const {Cu} = require("chrome");
 Cu.import("resource://gre/modules/ClientID.jsm");
 Cu.import("resource://gre/modules/Task.jsm");
 
+const PING_ENDPOINT = "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream";
```

Comment:
I agree, we should set it in `package.json` as a default pref.

e.g.

```javascript
  "preferences": [
    {
      "name": "telemetry.ping.endpoint",
      "title": "Telemetry server endpoint",
      "type": "string",
      "value": "https://onyx_tiles.stage.mozaws.net/v3/links/activity-stream"
    }
  ]
```

Here's one can use `simplePrefs`:

```javascript
const simplePrefs = require("sdk/simple-prefs");

simplePrefs.prefs["telemetry.ping.endpoint"];
```

For development, we probably want to give that pref another value. We should use the `dev-prefs.json` config file.
JPM commands for test and for command line use (`npm run once` etc.) should use `dev-prefs.json`.

e.g.
`$ jpm run --prefs ./dev-prefs.json`

We should amend existing npm scripts to make that work. The contents of `dev-prefs.json` could be the staging server.

@ncloudioj what would be the two endpoints we need here for staging and prod?